lib/ProxySQL_Statistics.cpp

```diff
@@ -789,7 +793,11 @@ void ProxySQL_Statistics::MyHGM_Handler_sets_v1(SQLite3_result *resultset) {
 
 void ProxySQL_Statistics::MySQL_Threads_Handler_sets(SQLite3_result *resultset) {
 	MySQL_Threads_Handler_sets_v1(resultset);
+#ifdef DEBUG
+	if (true) {
```

Comment:
We should add a comment that mentions we enable this feature in debug mode even if web interface is not enabled, for debugging and testing purpose.
Without comment we will forget (especially me).